source/kernel/modules.go

```diff
@@ -42,3 +44,35 @@ func getLoadedModules() ([]string, error) {
 	}
 	return loadedMods, nil
 }
+
+func getBuiltinModules() ([]string, error) {
+	kVersion, err := getVersion()
+	if err != nil {
+		return []string{}, err
+	}
+
+	kBuiltinModPath := hostpath.LibDir.Path("modules/" + kVersion + "/modules.builtin")
+	out, err := os.ReadFile(kBuiltinModPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read file %s: %s", kBuiltinModPath, err.Error())
+	}
+
+	lines := strings.Split(string(out), "\n")
+	loadedMods := make([]string, len(lines))
```

Comment:
nit: they're not "loaded" (like dynamically) so I'd rename this just `mods` or then `builtinMods`